docs/content/docs/cd/QApps.md

```diff
@@ -34,17 +33,16 @@ type: docs
 
 {{< /tab >}}
 
-{{< tab "Из CI" >}}
-Запусти конфигурацию [fullCheck](http://links.k.avito.ru/tmctAvitoAndroidFullCheck)\
-Выбери кастомный запуск, укажи свою ветку.
+{{< tab "CI" >}}
 
-## Known issues
+Run [uploadArtifacts](http://links.k.avito.ru/Mx9) Teamcity configuration.\
+Choose your custom branch if needed.
 
-**Длится до 1 часа**, запускает все тесты.\
-Чтобы ускорить, закомментируй лишние [шаги]({{< ref "/docs/ci/CIGradlePlugin.md#steps" >}}) в `fullCheck` конфиге в `build.gradle` приложений.\
-Задача на более быстрый способ: [MBS-7340](http://links.k.avito.ru/MBS7340)
 {{< /tab >}}
 {{< /tabs >}}
 
+## How to change upload settings?
```

Comment:
Should we define what users can change?

Reply:
good idea, I'll add it

Reply:
I've changed the phase a bit and decided not to duplicate documentation of this build step here.
To explain fully we need to explain artifacts.